### ISSUE TITLE
Filling Similar Colors in Fill Tool

### ISSUE THREAD

**Issue by ParodyVerse** [ZMYaro/paintz#107]
I would like to be able to fill a color with the fill tool and have it fill all the colors that are almost exactly the same, like in a JPG selecting all of a color, not just a random part of it.

**Comment by ParodyVerse** [comment]
PS
this would make it so much easier to get rid of a background image.

**Comment by ZMYaro** [comment]
@ParodyVerse: Thanks for the feature request.  Out of curiosity, is this a feature that exists in MS Paint?

**Comment by ParodyVerse** [comment]
Not that I know of. It would just be so much easier to fill JPGs.

**Comment by ZMYaro** [comment]
Yep, definitely, but it still takes dev time, so I will keep this ticket open, because I agree it would be useful, but I will probably not immediately prioritize it over MS Paint features.

**Comment by ParodyVerse** [comment]
Okay, thanks!
